src/types/Error.ts

```diff
@@ -0,0 +1,4 @@
+export type Error = {
+  title: string,
+  isDanger: boolean,
```

Comment:
Rename 'isDanger' to 'isError' or another suitable name to indicate that the property represents an error state.

Reply:
https://github.com/mate-academy/react_todo-app-loading-todos/pull/480/files#r1187301818
not fixed